www_src/pages/editor/text-editor.jsx

```diff
@@ -0,0 +1,93 @@
+var React = require('react/addons');
+var render = require('../../lib/render.jsx');
+var Binding = require('../../lib/binding.jsx');
+
+var Range = require('../../components/range/range.jsx');
+var ColorGroup = require('../../components/color-group/color-group.jsx');
+var {CheckboxSet, Radio} = require('../../components/option-panel/option-panel.jsx');
+
+var TextEditor = React.createClass({
+  mixins: [React.addons.LinkedStateMixin, Binding],
+  getInitialState: function () {
+    return {
+      fontSize: 18,
+      fontFamily: 'Roboto',
+      color: '#645839',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      textDecoration: 'none',
+      textAlign: 'center'
+    };
+  },
+  render: function () {
+    var textStyleOptions = [
+      {
+        id: 'fontWeight',
+        icon: '../../img/B.svg',
+        uncheckedValue: 'normal',
+        checkedValue: 'bold'
+      },
+      {
+        id: 'fontStyle',
+        icon: '../../img/I.svg',
+        uncheckedValue: 'normal',
+        checkedValue: 'italic'
+      },
+      {
+        id: 'textDecoration',
+        icon: '../../img/U.svg',
+        uncheckedValue: 'none',
+        checkedValue: 'underline'
+      }
+    ];
+    var textAlignOptions = [
+      {
+        value: 'left',
+        icon: '../../img/align-left.svg',
+      },
+      {
+        value: 'center',
+        icon: '../../img/align-center.svg',
+      },
+      {
+        value: 'right',
+        icon: '../../img/align-right.svg',
+      }
+    ];
+    return (
+      <div id="editor">
+        <div className="editor-preview">
+          <p style={this.state}>Hello world</p>
```

Comment:
this still feels like it needs to beunlazy'd - for instance, right now fontSize will be an illegal value, because it's a number without a unit.